ko/lessons/specifics/ets.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: page
+title: Erlang Term Storage (ETS)
+category: specifics
+order: 4
+lang: ko
+---
+
+Erlang Term Storage, 줄여서 ETS는 OTP 내부에 포함된 강력한 저장용 엔진으로, Elixir에서 이용할 수 있습니다. 이 레슨에서는 ETS를 호출하는 방법과 애플리케이션에서 사용하는 방법에 관해서 설명합니다.
+
+{% include toc.html %}
+
+## 개요
+
+ETS는 Elixir나 Erlang에서 객체를 메모리에 담아둘 수 있는 견고한 저장소입니다. ETS는 무척 큰 데이터를 저장할 수 있으며, 상수 시간 접근을 제공합니다.
+
+ETS의 테이블은 각각의 프로세스에 의해서 생성, 소유됩니다. 테이블을 소유하는 프로세스가 죽으면, 테이블도 함께 파괴됩니다. 초기 상태에서 ETS는 노드마다 1400개까지로 제한되어 있습니다.
+
+## 테이블 생성
+
+테이블은 `new/2`로 생성합니다. 이 함수는 테이블 이름과 옵션을 받으며, 뒤에서 설명할 함수들에서 사용할 수 있는 테이블 ID를 돌려줍니다.
+
+예를 들어 사용자를 이름으로 관리하고 검색하는 테이블을 만들어 보죠:
+
+```elixir
+iex> table = :ets.new(:user_lookup, [:set, :protected])
+8212
+```
+
+GenServer와 마찬가지로, ID 대신에 이름을 사용해서 ETS 테이블에 접근하는 방법이 있습니다. `:named_table` 옵션을 사용하면 이름을 통해 테이블에 접근할 수 있습니다:
+
+```elixir
+iex> :ets.new(:user_lookup, [:set, :protected, :named_table])
+:user_lookup
+```
+
+### 테이블의 타입
+
+ETS에서 사용할 수 있는 테이블의 타입은 4개가 있습니다:
+
++ `set` — 기본 테이블 타입입니다. 각 키당 하나의 값을 가지며, 키는 중복될 수 없습니다.
++ `ordered_set` — `set`과 비슷합니다만, Erlang/Elixir의 용어로 정렬할 수 있습니다. 중요하므로 추가로 설명하자면, 키의 비교는 `ordered_set`에서의 그것과 다릅니다. 
```

Comment:
`Keys need not match so long as they compare equally. 1 and 1.0 are considered equal.` need to be translated. This is placed at end of the sentence.